frontend/src/Component/Chat/ChatList.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Typography, Card, CardContent, CardMedia } from "@mui/material";
+import SearchIcon from "@mui/icons-material/Search";
+import { TempChat } from "../../Util/TempChat";
+import { DateConvert } from "../../Util/Date";
+
+const ChatList: React.FC = () => {
+  return (
+    <Box className="chatlist-container">
+      <Box className="chatlist-top">
+        <Typography color="inherit" className="chatlist-header">
+          대화목록
+        </Typography>
+        <SearchIcon className="search-icon"></SearchIcon>
+      </Box>
+      <Box className="chatlist">
+        {TempChat.map((item, idx) => (
```

Comment:
이 부분 내부 리턴값을 컴포넌트로 빼서 가독성을 올려도 좋을 것 같아요!

```suggestion
{TempChat.map((item, idx) => <Custom />}
```